parcels/particle.py

```diff
@@ -180,6 +181,7 @@ def __init__(self, size, grid, pclass=JITParticle,
         self.particles = np.empty(size, dtype=pclass)
         self.ptype = ParticleType(pclass)
         self.kernel = None
+        self.time_origin = grid.U.time_origin
```

Comment:
Do we need to store this on the particle set if we always pull `time_origin` from the field?